src/utils/index.js

```diff
@@ -0,0 +1,77 @@
+import {
+    SEPERATOR_STRING,
+    DEFAULT_SEPERATOR,
+    NUMBER_PATTERN,
+    SEPERATOR_SUBTITUTE,
+    NULL_STRING,
+    ERROR_MESSAGE,
+    EMPTY_INPUT_ARRAY
+} from '../constant/index.js'
+
+function getSeperatorArray(expression){
+    const seperator = DEFAULT_SEPERATOR;
+    if(hasCustomSeperator(expression)){
+        const seperatorStartIndex = expression.indexOf(SEPERATOR_STRING.START) + 2;
+        const seperatorEndIndex = expression.indexOf(SEPERATOR_STRING.END);
+        seperator.push(expression.slice(seperatorStartIndex, seperatorEndIndex))
+    }
+    return seperator;
+}
+
+function getExpressionString(expression){
+    if(hasCustomSeperator(expression)){
+        const expressionStartIndex = expression.indexOf(SEPERATOR_STRING.END) + 2;
+        return expression.slice(expressionStartIndex);
+    }
+    return expression;
+}
+
+function hasCustomSeperator(expression){
+    return expression.startsWith(SEPERATOR_STRING.START) && expression.includes(SEPERATOR_STRING.END);
+}
+
+function hasNullSeperatorError(seperatorArray){
```

Comment:
```suggestion
function hasBlankSeperatorError(seperatorArray){
```
여기서 `Null`은 `''` 빈 공백을 의미하는 것 같은데, 실제로 `null`이라는 값이 따로 있기 때문에 `blank`나 `empty`를 사용하는게 더 목적성이 드러날 수 있을 것 같은데 어떻게 생각하시나요??

Reply:
blank나 empty를 사용하면 훨씬 명확한 변수명이 될 것 같아요..! 리뷰 감사합니다 :)